test/action_view/integration_test.rb

```diff
@@ -35,6 +35,24 @@ class IntegrationTest < ActionDispatch::IntegrationTest
     HTML
   end
 
+  test "rendering component with content_for" do
+    get "/content_areas"
+    assert_response :success
+
+    expected_string = %(
+    <div>
+        <h1>Hi!</h1>
+
+        <p>Have a great #{Date.today.strftime("%A")}.</p>
```

Comment:
I noticed you changed this line from `<p>Have a great day.</p>` in c0e5f727. What was the motivation for making the assertion a bit more dynamic?

I know it doesn't matter in this case, but comparing the result of calling `Date.today` in a test and in an integration environment is setting off 🚨Time-Zone-mismatch alarms 🚨 in my head.

Reply:
The idea was to ensure we could render erb escaped syntax inside the `with` blocks. Your right that we could pick a simpler example